### PR TITLE
Fixes: Better DeleteDevice and always read state from NI

### PR DESCRIPTION
**Pull request by tiagocoutinho**
This introduces two fixes:
* Make sure all objects are destroyed on DeleteDevice
* Make sure we read state from NI device and not from cache

### CODE REVIEW

File: sardana_ni660x/ctrl/Ni660XCTCtrl.py [hunk 1]
```diff
@@ -211,13 +211,14 @@ def DeleteDevice(self, axis):
             self.channels[axis].removeListener(self.counterEventReceived)
             self.attributes.pop(axis)
             self.ch_configured.pop(axis)
-        self.channels.pop(axis)
+        ch = self.channels.pop(axis)
```
Comment (reszelaz) [inline sardana_ni660x/ctrl/Ni660XCTCtrl.py]: I think this assignment to a variable is not necessary. We don't use it afterwards, right?

**Review by reszelaz — COMMENTED** [review]
LGTM, just one tiny comment.

File: sardana_ni660x/ctrl/Ni660XCTCtrl.py (hunk 1 above)
Comment (tiagocoutinho) [reply sardana_ni660x/ctrl/Ni660XCTCtrl.py]: Yes, fixed now. Thanks!

**Comment by tiagocoutinho** [comment]
I added a new fix (4ef1a0a and dcb630a) which avoids subscribe/unsubscribe to tango config events each time an attribute needs to be read from the NI tango server

**Pull request merged**